NEWS.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.6 2018-03-05
+ * Remove the RM API query argument "selection=selected" as the default when
+   the "ext.selected" query argument is not passed to the codex API. The UI has
+   been updated to not pass "ext.selected" which will mean all, otherwise it
+   passes "ext.selected=true" or "ext.selected=false" to represent the RM API
+   query arguments "selection=selected" and "selection=notselected"
+   respectively. Passing "ext.selected=all" will explicitly pass "selction=all"
```

Comment:
a minor comment: typo of selection in this line

Reply:
Oops. :)